build/Microsoft.DotNet.Cli.Run.targets

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Are the clean targets in BuildTools not extensible enough to avoid these custom clean targets?